lib/mapper.js

```diff
@@ -55,16 +55,43 @@ exports.track = function(track, settings) {
   return ret;
 };
 
-function normalize(event) {
-  var normalizedEvent = { };
-  for (var name in event) {
-    if (event.hasOwnProperty(name)) {
-      normalizedEvent[normalizeName(name)] = event[name];
+function normalize(value) {
+  if (value === undefined) {
+    return undefined;
+  }
+  if (value === null) {
+    return null;
+  }
+  if (value instanceof Array) {
```

Comment:
Doesn't look like we're adding a test case for this. Any chance you could add one?